### ISSUE TITLE
Personal Nanobot Beacon Not Working Despite Chunk Loaded

### ISSUE THREAD

**Issue by AzzBlock** [ValkyrieofNight/EnvironmentalTech#178]
Hi. I have built a Personal Nanobot Beacon which works fine when I am within range of it but if I fly to far away or go to another dimension it stops working despite being chunk loaded. 

I have tried several different ways of chunk loading the beacon and none of them work. However the chunk loaders are definitely working as I can open up my AE2 wireless terminal and see items going into the system from various farms which are also chunk loaded.

First time using this mod and otherwise I have not had any other problems. I am using the mod on Modern Skyblock Departed 3.

**Comment by AzzBlock** [comment]
To anyone who might be encountering this problem I have found a fix. I connected a timer (any redstone clock will do the job) with a 40 tick delay (longer delays might work I have not experimented) to the Nanobot beacon meaning it sends a short pulse which turns it off very briefly every couple of seconds. The pulse is short enough that it doesn't cause a loss of any of the beacons effects (I know this because I have just been flying around the nether with no issues at all using the Creative Flight Modifier). 

Specifically I used RFtools Timer for sending the redstone signal and also attached it to a PneumaticCraft Universal Sensor which is set to detect the player over a large range. This turns off the timer when I am in range of the beacon but turns on if a fly out of range or go to another dimension. Of course for the redstone to work when you are not there the area must be chunk loaded.
